components/Navigation.js

```diff
@@ -0,0 +1,55 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const Navigation = () => {
+  const router = useRouter();
+
+  return (
+    <NavBar>
+      <NavItem $active={router.pathname === "/"}>
+        <Link href="/">I</Link>
+      </NavItem>
+    </NavBar>
+  );
+};
+
+export default Navigation;
+
+const NavBar = styled.nav`
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 90%;
+  border-radius: 25px;
+  display: flex;
+  justify-content: space-around;
+  background-color: #fff;
+  box-shadow: 0px -1px 5px rgba(0, 0, 0, 0.1);
+  padding: 10px 0;
+`;
+
+const NavItem = styled.div`
```

Comment:
In combination with my suggestion from above:

```suggestion
const StyledLink = styled(Link)`
```